internal/pkg/cli/task_run.go

```diff
@@ -31,11 +31,14 @@ var (
 )
 
 var (
+	fmtTaskRunAppPrompt = fmt.Sprintf("In which %s would you like to run this %s?", color.Emphasize("application"), color.Emphasize("task"))
 	fmtTaskRunEnvPrompt       = fmt.Sprintf("In which %s would you like to run this %s?", color.Emphasize("environment"), color.Emphasize("task"))
 	fmtTaskRunGroupNamePrompt = fmt.Sprintf("What would you like to %s your task group?", color.Emphasize("name"))
 
+	taskRunAppPromptHelp = fmt.Sprintf("Task will be deployed to the selected application. " +
+		"Select %s to run the task in your default VPC instead of any existing application.", color.Emphasize(appEnvOptionNone))
```

Comment:
If possible, let's try to limit the text shown on the screen to be less than 80 chars as that's the default char width when you open the terminal. (see https://github.com/aws/copilot-cli/blob/master/STYLE_GUIDE.md#screen-sizes)
```suggestion
	taskRunAppPromptHelp = fmt.Sprintf(`Task will be deployed to the selected application. 
Select %s to run the task in your default VPC instead of any existing application.`, color.Emphasize(appEnvOptionNone))
```